src/components/idleUsers/section/index.tsx

```diff
@@ -1,21 +1,15 @@
 import { FC } from 'react';
-import Card from '@/components/idleMembers/card';
-import classNames from '@/components/idleMembers/section/section.module.scss';
+import Card from '@/components/idleUsers/card';
+import classNames from '@/components/idleUsers/section/section.module.scss';
+import { idleUserArray, idleUser } from '@/interfaces/idleUser.type';
 
-type Props = {
-  heading: string
-  content: Array<string>
-  error: string | null
-  isLoading: boolean
-}
-
-function renderCards(content: Array<string>) {
-  return content.map((idleMember) => (
-    <Card idleMemberUserName={idleMember} key={idleMember} />
+function renderCards(content: idleUser[]) {
+  return content.map((idleUser) => (
+    <Card user={idleUser} key={idleUser.id} />
   ));
 }
```

Comment:
Should this be not wrapped inside a fragment?

Reply:
I don't think so. Wrapping is necessary when you have multiple tags